imports/ui/pages/compareCurrencies/compareCurrencies.js

```diff
@@ -110,21 +110,22 @@ Template.compareCurrencies.events({
 		  		_id: 'elodata'
 		  	}) || {}
 
-		  	let wallet = currency.walletRanking / graphdata.walletMaxElo * 10
 
-		  	let community = currency.communityRanking / graphdata.communityMaxElo * 10
+			const {codebaseMaxElo, codebaseMinElo, communityMaxElo, communityMinElo, walletMinElo, walletMaxElo} = graphdata
 
-		  	let codebase = (currency.codebaseRanking || 400) / graphdata.codebaseMaxElo * 10
+			var wallet = ((currencyData.walletRanking - walletMinElo)/((walletMaxElo - walletMinElo) || 1)) * 10;
+			var community = (((currencyData.communityRanking || communityMinElo) - communityMinElo) / ((communityMaxElo - communityMinElo) || 1)) * 10;
+			let codebase = (((currencyData.codebaseRanking || codebaseMinElo) - codebaseMinElo) / ((codebaseMaxElo - codebaseMinElo) || 1)) * 10
 
 		  	let maxD = graphdata.decentralizationMaxElo
 		  	let minD = graphdata.decentralizationMinElo
 
-		  	let decentralization = (((currency.decentralizationRanking || 400) - minD) / ((maxD - minD) || 1)) * 10 
+		  	let decentralization = (((currencyData.decentralizationRanking || minD) - minD) / ((maxD - minD) || 1)) * 10 

```

Comment:
If I'm not mistaken, `currencyData` is not defined in this scope. (only `currency` is defined)